config/lint/lint-config.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<lint>
+
+    <issue id="all">
+        <ignore path="build" />
+    </issue>
+
+    <!-- Remove this 'UnusedResources' when Lint for Kotlin is updated
+    Android Lint: False-positive Lint warning "unused resources" for resources in Kotlin code
+    https://youtrack.jetbrains.com/issue/KT-12499 -->
+    <issue id="UnusedResources" severity="ignore" />
```

Comment:
Yes. Lint doesn't work well with Kotlin yet. It reports all strings and drawables as unused even though they are referenced in .kt files.